modules/auxiliary/admin/mssql/mssql_exec_oacreate.rb

```diff
@@ -0,0 +1,40 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::MSSQL
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Microsoft SQL Server sp_oacreate Command Execution',
+      'Description'    => %q{
+      This module will execute a Windows command on a MSSQL/MSDE instance
+      via the sp_oacreate procedure (ole) instead of the xp_cmdshell. A valid username and password is required
+      to use this module
```

Comment:
Slightly nicer formatting:

```suggestion
        This module will execute a Windows command on a MSSQL/MSDE instance
        via the sp_oacreate procedure (ole) instead of the xp_cmdshell.
        A valid username and password is required to use this module.
```